lib/avo/concerns/fetches_things.rb

```diff
@@ -51,6 +51,9 @@ def get_resource_by_name(name)
         #
         # get_resource_by_name('User') => UserResource
         # get_resource_by_name(User) => UserResource
+
+        # FIXME: This query doesn't necessarily get the right resource.
+        # From all the resources that use the same model_class, it will fetch the first one in alphabetical order
```

Comment:
Could we make the whole comment block like so?

```ruby
        # Returns the Avo resource by singular snake_cased name
        # From all the resources that use the same model_class, it will fetch the first one in alphabetical order
        #
        # get_resource_by_name('User') => UserResource
        # get_resource_by_name(User) => UserResource
```

The method isn't really broken. It will still fetch the resource as requested. If the resource is the wrong one, that needs to be fixed in its proper place.